Zend/zend_exceptions.stub.php

```diff
@@ -22,6 +22,13 @@ public function getTraceAsString(): string;
 
 class Exception implements Throwable
 {
+    /** @known */
+    protected $message = "";
+    private $string = "";
+    protected $code = 0;
+    protected $file = null;
```

Comment:
Unlike in userland code, internal properties are either declared as `null` or `undefined`, so I wanted to keep this distinction by adding the explicit `null` default value.

Reply:
The previous comment was simply a lie :) It turned out that only typed properties are registered with an `undefined` default value